src/Provider/IP2Location/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2019 — IP2Location <support@ip2location.com>
```

Comment:
Please keep the original license!

Reply:
I had a quick look and a lot of new providers seem to have the author in them. Wonder if we have guidelines around this (or should we?).
https://github.com/geocoder-php/Geocoder/blob/master/src/Provider/Mapbox/LICENSE
https://github.com/geocoder-php/Geocoder/blob/master/src/Provider/AlgoliaPlaces/LICENSE

Most of the time we set the copyright blocks to the new author as well.

Reply:
Oh okay, nevermind then! :)